lib/Server.js

```diff
@@ -95,6 +95,7 @@ class Server {
     this.hot = this.options.hot || this.options.hotOnly;
     this.headers = this.options.headers;
     this.progress = this.options.progress;
+    this.liveReload = this.options.liveReload;
```

Comment:
Don't add here new options, you can see https://github.com/webpack/webpack-dev-server/blob/0523cfd0309f7add440169b523176ed7baa64ec3/lib/Server.js#L94 use `this.options.liveReload` everywhere